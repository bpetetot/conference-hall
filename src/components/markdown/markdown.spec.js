/* eslint-env jest */
import snap from 'tests/snapshot'
import Markdown, { MarkdownIcon } from './index.js'

const snapshot = props => snap(Markdown)({ ...props })
const snapshotIcon = props => snap(MarkdownIcon)({ ...props })

describe('components/mardown', () => {
  it('should not render if no source', snapshot({}))
  it('should render', snapshot({ source: 'source' }))
  it('should render icon', snapshotIcon({ className: 'c' }))
})
