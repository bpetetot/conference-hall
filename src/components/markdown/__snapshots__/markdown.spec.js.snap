// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/mardown should not render if no source 1`] = `
<div
  className="markdown"
  dangerouslySetInnerHTML={
    Object {
      "__html": "",
    }
  }
/>
`;

exports[`components/mardown should render 1`] = `
<div
  className="markdown"
  dangerouslySetInnerHTML={
    Object {
      "__html": "source",
    }
  }
/>
`;

exports[`components/mardown should render icon 1`] = `
<svg
  className="c"
  viewBox="0 0 208 128"
  xmlns="http://www.w3.org/2000/svg"
>
  <rect
    height="118"
    ry="10"
    strokeWidth="10"
    width="198"
    x="5"
    y="5"
  />
  <path
    d="M30 98v-68h20l20 25 20-25h20v68h-20v-39l-20 25-20-25v39zM155 98l-30-33h20v-35h20v35h20z"
  />
</svg>
`;
